examples/gallery/gallery.pro

```diff
@@ -45,6 +45,12 @@ SOURCES += \
 HEADERS += \
     Page.h
 
+HEADERS += \
+    slider/LinearGradientSlider.h \
+
+SOURCES += \
+    slider/LinearGradientSlider.cpp \
+
```

Comment:
@uwerat maybe move this control to the `tabview` example? Or maybe even an individual example `skinning` ?

Reply:
Please put the gradient slider to the tabview example.

At the moment the tabview example is a bit naked, but I can imagine to turn it into some sort of playground for custom controls. F.e the one from mycontrols would fit here too.

Reply:
done